npm-shrinkwrap.json

```diff
@@ -97,6 +97,9 @@
         "ts-jest": "^27.1.4",
         "ts-node": "^10.7.0",
         "typescript": "^4.6.3"
+      },
+      "engines": {
+        "node": ">=14"
```

Comment:
Followed remix package.json here.

Reply:
great, we didn;t have the enginers before -- good to add it 👍🏼